haystack/nodes/summarizer/transformers.py

```diff
@@ -122,10 +119,10 @@ def __init__(
     def predict(self, documents: List[Document]) -> List[Document]:
         """
         Produce the summarization from the supplied documents.
-        These document can for example be retrieved via the Retriever.
+        The documents can come from the Retriever.
 
-        :param documents: Related documents (e.g. coming from a retriever) that the answer shall be conditioned on.
-        :return: List of Documents, where Document.meta["summary"] contains the summarization
+        :param documents: Related documents (for example, coming from a Retriever) that the answer is conditioned on.
```

Comment:
`Related documents (for example, coming from a Retriever) that the answer is conditioned on.` sounds like retrieval augmented generation. Instead, we should say sth like `List of Documents to summarize individually`. `related documents` and `answers` doesn't fit here for the summarizer.